_data/guides.yaml

```diff
@@ -7,9 +7,41 @@ categories:
       - title: Getting Started with WildFly
         url: /get-started
         description: Build and run a Jakarta EE application with WildFly in a few minutes
-      - title: Java Microservices on Kubernetes with WildFly
-        url: /guides/get-started-microservices-on-kubernetes
-        description: Build and run Jakarta EE applications with WildFly on Kubernetes in a few minutes
+  - category: Cloud / Containerization
+    cat-id: docker-kubernetes
```

Comment:
Can we get rid of `docker` also from the category id?

Reply:
@rhusar Tnks, changed